panel/applets/status/StatusApplet.vala

```diff
@@ -92,6 +108,7 @@ public class StatusApplet : Budgie.Applet
     {
         this.manager = manager;
         manager.register_popover(blue.ebox, blue.popover);
+        manager.register_popover(power.ebox, power.popover);
```

Comment:
You've registered the wrong widget. The popover is connected to a different widget than the one you're registering with